modules/tiles/test/tileset/tileset-3d.spec.js

```diff
@@ -154,7 +154,7 @@ const VIEWPORTS = [
 ];
 
 test('Tileset3D#throws with undefined url', (t) => {
-  t.throws(() => new Tileset3D());
+  t.throws(() => new Tileset3D(undefined));
```

Comment:
You should need a `@ts-ignore` here, otherwise type checking seems broken.

Reply:
Done